testing/e2e_docs.md

```diff
@@ -0,0 +1,53 @@
+## Arlon e2e tests
```

Comment:
can you rename the file to `e2e_testing.md` or similar?

Reply:
Sure